data/beps/beps.yml

```
@@ -597,3 +597,33 @@
     google_doc_created: 2023-09-12
     pull_request_created:
     pull_request_merged:
+
+
+-   number: '046'
+    title: Cardiac MRI (CMR)
+    google_doc: [[https://docs.google.com/document/d/1StWtkrQpJyBjPcPjAMrk07QJcoG2gGzih8LX0lhmGKI/edit]]
```

```suggestion
    google_doc: [[https://docs.google.com/document/d/1FmE1VcPtUIh4uuJWY0WFGvGWVjjnbNXc/edit]]
```
Updating the Googledoc link, March 10, since the old one was not editable by key people.